src/quantuminspire/api.py

```diff
@@ -676,6 +676,32 @@ def get_measurement_register_from_result(self, result_id: int) -> List[Any]:
             raise ApiError(f'Measurement register for result with id {result_id} does not exist!') from err_msg
         return measurement_register
 
+    def get_calibration_from_result(self, result_id: int) -> Optional[Any]:
+        """ Gets the calibration information of the executed cQASM code, given the result_id.
```

Comment:
```suggestion
        """ Gets the calibration information of the executed code, given the result_id.
```
We also submit qiskit code (right that that is converted to cqasm internally, but that might change)

Reply:
Changed it to executed job. Also changed some other instances of cQASM (where it is not needed to be called cQASM explicitly)